go/pvl/interp.go

```diff
@@ -1011,9 +1011,9 @@ func runSelectorJSONInner(m metaContext, state scriptState, selectedObject *json
 	logger := func(format string, args ...interface{}) {
 		debugWithState(m, state, format, args)
 	}
-	jsonResults, perr := jsonhelpers.AtSelectorPath(selectedObject, selectors, logger)
-	if perr != nil {
-		return nil, perr
+	jsonResults, perr := jsonhelpers.AtSelectorPath(selectedObject, selectors, logger, libkb.NewInvalidPVLSelectorError)
+	if perrInner, _ := perr.(libkb.ProofError); perrInner != nil {
```

Comment:
im worried that if there's a `perr` that does not satisfy the if condition, then it will not branch here and instead continue in bad state